enclave-runtime/Cargo.lock

```diff
@@ -3543,7 +3543,21 @@ dependencies = [
 [[package]]
 name = "sgx_build_helper"
 version = "1.1.6"
-source = "git+https://github.com/apache/incubator-teaclave-sgx-sdk?branch=v1.1.6-testing#9c1bbd52f188f600a212b57c916124245da1b7fd"
+source = "git+https://github.com/apache/incubator-teaclave-sgx-sdk?branch=master#27bd225ae6dbcd1d0a6d4d9590acc4d73c5195c2"
+
+[[package]]
+name = "sgx_crypto_helper"
+version = "1.1.6"
+source = "git+https://github.com/apache/incubator-teaclave-sgx-sdk?branch=master#27bd225ae6dbcd1d0a6d4d9590acc4d73c5195c2"
+dependencies = [
+ "itertools",
+ "serde 1.0.118 (git+https://github.com/mesalock-linux/serde-sgx)",
+ "serde-big-array",
+ "serde_derive 1.0.118",
+ "sgx_tcrypto",
+ "sgx_tstd",
+ "sgx_types",
+]
```

Comment:
Yes, as you can see below, we still import the crypto helper from the testing branch too.